velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -850,5 +854,19 @@ TEST_F(StringTest, findInSet) {
   EXPECT_EQ(
       findInSet("abåæçè", "\u002c\u0061\u0062\u00e5\u00e6\u00e7\u00e8"), 2);
 }
+
+TEST_F(StringTest, initCap) {
+  EXPECT_EQ(
+      initcap("àáâãäåæçèéêëìíîïðñòóôõöøùúûüýþ"),
+      "Àáâãäåæçèéêëìíîïðñòóôõöøùúûüýþ");
+  EXPECT_EQ(initcap("αβγδεζηθικλμνξοπρςστυφχψ"), "Αβγδεζηθικλμνξοπρςστυφχψ");
+  EXPECT_EQ(initcap("hello world"), "Hello World");
```

Comment:
Could you add a test with mixed unicode & ascii? E.g., some word with ascii-encoded initial character, some are not.